scripts/coreos-ipa-common.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/bash
+
+ROOTFS_FILE=/shared/html/images/ironic-python-agent.rootfs
+IGNITION_FILE=/shared/html/ironic-python-agent.ign
+ISO_FILE=/shared/html/images/ironic-python-agent.iso
```

Comment:
Can we make this configurable via argument to the script instead of a hardcoded FFILENAME?

Reply:
I'm actually not very particular about the FFILENAME here, but it does make sense that the name is `ironic-python-agent` because its a intermediate image. I'll apply the name change to the rhcos-downloader image and test them together.

Reply:
I think not supporting custom names here will make the diff smaller with the upstream version (which does hardcode ironic-python-agent).

Reply:
So...is the plan to still use the openshift fork of ironic? This has implications in the installer's startironic.sh script because the way ironic and its dependencies are started varies in metal3 vs. openshift versions.

Reply:
The openshift fork of ironic will be re-synced with metal3 soon(ish), once Riccardo comes back from PTO.

Reply:
there it is https://github.com/openshift/ironic-image/pull/190